src/mixins/canvas_events.mixin.js

```diff
@@ -712,8 +712,8 @@
       if (this.selection && (!target ||
         (!target.selectable && !target.isEditing && target !== this._activeObject))) {
         this._groupSelector = {
-          ex: pointer.x,
-          ey: pointer.y,
+          ex: this._absolutePointer.x,
+          ey: this._absolutePointer.y,
```

Comment:
i dream a day in which the local pointer goes away entirely and fabric uses only the absolute values, and we remove most of the conversions.